Yoshi/Yoshi/YoshiConfigurationManager.swift

```diff
@@ -34,39 +34,39 @@ internal final class YoshiConfigurationManager {
             return
         }
 
-        let navigationController = UINavigationController()
-        let debugViewController = DebugViewController(options: yoshiMenuItems, completion: { [weak self] in
-            self?.animate({
-                    self?.presentingWindow?.alpha = 0.0
-                }, completed: {
-                    self?.presentingWindow = nil
-            })
-        })
-
-        navigationController.setViewControllers([debugViewController], animated: false)
-
-        let window = UIWindow()
-        window.rootViewController = navigationController
+        let window = UIWindow(frame: UIScreen.mainScreen().bounds)
         window.windowLevel = UIWindowLevelNormal
-        presentingWindow = window
 
+        // Use a dummy view controller with clear background.
+        // This way, we can make the actual view controller we want to present a form sheet on the iPad.
+        let rootViewController = UIViewController()
+        rootViewController.view.backgroundColor = UIColor.clearColor()
+
+        window.rootViewController = rootViewController
         window.makeKeyAndVisible()
-        window.alpha = 0.0
-        window.rootViewController?.view.layoutIfNeeded()
-        animate {
-            window.alpha = 1
+        
+        presentingWindow = window
+        
+        // iOS doesn't like when modals are presented right away after a window's been made key and visible.
+        // We need to delay presenting the debug controller a bit to suppress the warning.
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, Int64(0 * Double(NSEC_PER_SEC))), dispatch_get_main_queue()) { [weak self] in
+            self?.presentDebugViewController()
```

Comment:
Do we need weak here? I feel like this block will be destroyed after it is executed (also it's a singleton), so we can be confident that it'll exist to handle this.